gravitee-apim-console-webui/src/portal/customization/api/api-list/portal-api-list.component.html

```diff
@@ -0,0 +1,149 @@
+          (cdkDropListDropped)="drop($event)"
+          cdkDropListData="dataSource"
+        >
+          <caption style="display: none">
+            Table with API Information
+          </caption>
+
+          <!-- Position Column -->
+          <ng-container matColumnDef="order" sticky>
+            <th mat-header-cell *matHeaderCellDef></th>
+            <td mat-cell *matCellDef="let element">
+              <mat-icon class="drag-cursor">drag_indicator</mat-icon>
+            </td>
+          </ng-container>
+
+          <!-- Name Column -->
+          <ng-container matColumnDef="name">
+            <th mat-header-cell *matHeaderCellDef>Name</th>
+            <td mat-cell *matCellDef="let element">{{ element.name }}</td>
+          </ng-container>
+
+          <!-- Value Column -->
+          <ng-container matColumnDef="value">
+            <th mat-header-cell *matHeaderCellDef>Value</th>
+            <td mat-cell *matCellDef="let element">{{ element.value }}</td>
+          </ng-container>
+
+          <!-- Actions Column -->
+          <ng-container matColumnDef="actions">
+            <th mat-header-cell *matHeaderCellDef id="actions"></th>
+            <td mat-cell *matCellDef="let element">
+              <div class="actions__edit">
+                <button
+                  (click)="updateHeader(element)"
+                  mat-icon-button
+                  aria-label="Button to edit header"
+                  matTooltip="Edit header"
+                  *gioPermission="{ anyOf: ['environment-api-c', 'environment-api-u', 'environment-api-d'] }"
+                  data-testid="edit-header-button"
+                >
+                  <mat-icon data-testid="api_list_edit_button">edit</mat-icon>
+                </button>
+
+                <button
+                  (click)="deleteHeader(element)"
+                  mat-icon-button
+                  aria-label="Delete header button"
+                  *gioPermission="{ anyOf: ['environment-api-d'] }"
+                  data-testid="delete-button"
+                >
+                  <mat-icon data-testid="api_list_edit_button">delete</mat-icon>
+                </button>
+              </div>
+            </td>
+          </ng-container>
+
+          <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+          <tr
+            mat-row
+            *matRowDef="let row; columns: displayedColumns"
+            data-testid="link_table_row"
+            cdkDrag
+            [cdkDragData]="row"
+            cdkDragLockAxis="y"
+          ></tr>
+
+          <!-- Row shown when there is no data -->
+          <tr class="mat-mdc-row mdc-data-table__row" *matNoDataRow>
+            <td class="mat-mdc-cell mdc-data-table__cell" [attr.colspan]="displayedColumns.length">No headers to display.</td>
+          </tr>
+        </table>
+      </mat-card>
+      <gio-save-bar [form]="form" [formInitialValues]="form" (resetClicked)="reset()" (submitted)="submit()"></gio-save-bar>
```

Comment:
`formInitialValues` should be the initial values of the form, not the form itself

Reply:
Done